src/index.html

```diff
@@ -7,12 +7,60 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__content">
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple_A1419"
+          class="card__picture"
+        />
+
+        <p class="card__product-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="stars">
+            <div class="stars">
+              <div class="stars__star"></div>
```

Comment:
Add the modifier class
```suggestion
            <div class="stars stars--4">
              <div class="stars__star"></div>
```